gp-post-content-merge-tags/gppcmt-save-confirmation-url-to-field.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * Gravity Perks // GP Post Content Merge Tags // Save Confirmation URL to Field
+ * https://gravitywiz.com/documentation/gravity-forms-post-content-merge-tags/
+ *
+ * This snippet saves the Confirmation URL to a field. 
```

Comment:
Update the description to:
> Save the confirmation URL (including the generated `eid` parameter) to a field on your form so that it can be mapped via Gravity Forms feeds and used anywhere field-based merge tags are supported.

Reply:
I've updated our snippet instructions to better explain what we're looking for in the snippet description. Let me know if you have any questions! 

https://www.notion.so/gravitywiz/Snippet-Library-Code-Organization-Formatting-and-Naming-d5bee101d9674dfc98d7ec007867de2b#38904efeb42b42e0ab662aa8adb7b18d

Reply:
The description has been updated and I have noted the instructions in the notion documentation.